### ISSUE TITLE
drush dl fails with wget --show-progress

### ISSUE THREAD

**Issue by molenick** [backdrop-contrib/drush#18]
Great to see some major progress on backdrop-drush! :)  I'm running into an issue where drush dl is failing like so:

```
matto@medusa:~/workspace/molenick$ drush dl devel
wget: unrecognized option '--show-progress'
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
unzip:  cannot find or open /home/matto/workspace/molenick/modules/devel.zip, /home/matto/workspace/molenick/modules/devel.zip.zip or /home/matto/workspace/molenick/modules/devel.zip.ZIP.
rm: cannot remove ‘/home/matto/workspace/molenick/modules/devel.zip’: No such file or directory
```

Using --show-progress is nice, but it looks like it isn't supported in my version of WGET.  I'm on Xubuntu 14.04, and here's the output of wget -V:

```
wget -V
GNU Wget 1.15 built on linux-gnu.

+digest +https +ipv6 +iri +large-file +nls +ntlm +opie +ssl/openssl 

Wgetrc: 
    /etc/wgetrc (system)
Locale: 
    /usr/share/locale 
Compile: 
    gcc -DHAVE_CONFIG_H -DSYSTEM_WGETRC="/etc/wgetrc" 
    -DLOCALEDIR="/usr/share/locale" -I. -I../../src -I../lib 
    -I../../lib -D_FORTIFY_SOURCE=2 -I/usr/include -g -O2 
    -fstack-protector --param=ssp-buffer-size=4 -Wformat 
    -Werror=format-security -DNO_SSLv2 -D_FILE_OFFSET_BITS=64 -g -Wall 
Link: 
    gcc -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat 
    -Werror=format-security -DNO_SSLv2 -D_FILE_OFFSET_BITS=64 -g -Wall 
    -Wl,-Bsymbolic-functions -Wl,-z,relro -L/usr/lib -lssl -lcrypto 
    -ldl -lz -lidn -luuid ftp-opie.o openssl.o http-ntlm.o 
    ../lib/libgnu.a 

Copyright (C) 2011 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later
<http://www.gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Originally written by Hrvoje Niksic <hniksic@xemacs.org>.
Please send bug reports and questions to <bug-wget@gnu.org>.
```

I'm going to fork, delete the the --show-progress flag from the wget call and see how that does.

**Comment by molenick** [comment]
I forked, removed the flags and now it works.  Not sure if there's a way to conditionally use --show-progress if it's available, but to me it seems like pretty trivial functionality (I can't think of instances where I've been waiting on drush dl to complete).

Putting in a pull request, do with it as you wish! :)

**Comment by serundeputy** [comment]
Thanks @molenick !
Thanks for your supportive words and for trying drush with backdrop.
I think this is a good change.  I'll pull it down this weekend test and if all is well merge.

Thanks for the `PR`!

~Geoff

**Comment by molenick** [comment]
Thanks, it's my pleasure.  I'd love to work on backdrop-drush more as I am able.  I'm not super-familiar with drush's internals, but I've used it a long time and have written a lot of custom drush commands.  Please let me know any way that would be useful for me to plug in.

Matt

**Comment by serundeputy** [comment]
That's great @molenick 

Many of the commands have to be rebuilt to work with backdrop; so if you pick your favourite one, make an issue here and in your spare time work on a `PR` for it.   For example it would be nice to have `drush pm-list`, but pick your favourite.

thanks again!
see you soon,
~Geoff

**Comment by serundeputy** [comment]
thanks @molenick merged!
